test/data/javascript/tests.js

```diff
@@ -230,7 +230,18 @@ exports.test_data = {
     }, {
       name: "Object literal shorthand functions",
       description: "Object literal shorthand functions",
-      tests: [
+      tests: [{
+          comment: "#1838 - handle class word as an object property",
+          unchanged: [
+            '{\n',
+            '    class: {\n',
+            '        a: 1,\n',
+            '        b: 2,\n',
+            '        c: 3,\n',
+            '    }\n',
+            '}'
```

Comment:
Please move your test input to the regression test area down around line 3500 or so.  This doesn't relate to "Object literal shorthand functions".

Reply:
I wasn't sure about this, thanks for letting me know. I have moved them under regression tests and also added more tests.